staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2869,7 +2869,7 @@ type InterfaceBindingPlugin struct {
 	// version: 1alphav1
 	NetworkAttachmentDefinition string `json:"networkAttachmentDefinition,omitempty"`
 	// DomainAttachmentType is a standard domain network attachment method kubevirt supports.
-	// Supported values: "tap".
+	// Supported values: "tap", "managedTap" (since v1.4).
```

Comment:
General question, is this ever validated?

Reply:
>I guess the disadvantage is that a user will not know why it is not working.
If this is a missing peace, then we can consider fixing it as a follow up.
>Do you have an opinion on this?

This works for me as it is "bug" of the past. I think at least for the primary we should validate.